core/control-panel/impl/src/models/registry.rs

```diff
@@ -12,21 +12,18 @@ use std::collections::BTreeMap;
 /// This value is hardcoded, and it should be updated or moved to a configurable value in the future if needed.
 pub const _UNIQUE_TAGS: [&str; 2] = ["latest", "stable"];
 
-/// The registry entry id, which is a UUID.
-pub type RegistryEntryId = UUID;
+/// The registry id, which is a UUID.
+pub type RegistryId = UUID;
 
-/// The registry entry is a record that is stored in the registry repository.
+/// The registry is a record that is stored in the registry repository.
 ///
 /// It stores entries about wasm modules and can be extended to other entry types. When adding new entry types,
-/// the `RegistryEntryValue` enum should be updated to include the new entry type.
+/// the `RegistryValue` enum should be updated to include the new entry type.
 #[storable]
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
-pub struct RegistryEntry {
+pub struct Registry {
```

Comment:
To me Registry sounds like a collection of items, not the item itself in the collection.

Reply:
Makes sense, i'll rename this back to `RegistryEntry`